app/services/base_service.rb

```diff
@@ -2,6 +2,8 @@
 
 # Base class for service classes
 class BaseService
+  include RequestErrorTrackingHelper
```

Comment:
`Helper`s are specifically for views in the context of rails apps, I don't like having them being included in services. Maybe you can move the `RequestErrorTrackingHelper` inside the `services` folder

Reply:
Great, thank you for the insight!